cflat.md

```diff
@@ -0,0 +1,100 @@
+> [name=Arbie, Yin, Guo, Ashima]
+
+## Information about the project
+
+### What is compiler explorer?
```

Comment:
I don't think you need to explain what Compiler Explorer is here. I think anyone looking in this repository will know.